### PR TITLE
Change soft delete for InstitutionStatus

### PR DESCRIPTION
**Pull request by alrom**

### CODE REVIEW

File: OutOfSchool/OutOfSchool.WebApi/Services/StatusService.cs [hunk 1]
```diff
@@ -96,19 +97,20 @@ public async Task<InstitutionStatusDTO> Update(InstitutionStatusDTO dto)
     {
         logger.LogInformation($"Updating InstitutionStatus with Id = {dto?.Id} started.");
 
-        try
-        {
-            var institutionStatus = await repository.Update(mapper.Map<InstitutionStatus>(dto)).ConfigureAwait(false);
-
-            logger.LogInformation($"InstitutionStatus with Id = {institutionStatus?.Id} updated succesfully.");
+        var institutionStatus = await repository.GetById(dto.Id);
```
Comment (VadymLevkovskyi) [inline OutOfSchool/OutOfSchool.WebApi/Services/StatusService.cs]: (Performance) Isn't `.ConfigureAwait(false)` missed here?

Comment (VadymLevkovskyi) [reply OutOfSchool/OutOfSchool.WebApi/Services/StatusService.cs]: (Functionality) I don't see here check of `IsDeleted`, and seems `Update` could work with both deleted and not deleted entities. As I don't see Update method calling places, so asking you to consider if `Update` should be used for deleted entities.

**Review by VadymLevkovskyi — COMMENTED** [review]
Looks good, please check comments

File: OutOfSchool/OutOfSchool.WebApi/Services/StatusService.cs (hunk 1 above)
Comment (alrom) [reply OutOfSchool/OutOfSchool.WebApi/Services/StatusService.cs]: 1. Done. I thought it isn't necessary for Core.
2. Done. I planned to use it for both options. We discussed on the daily and I fixed it.

Thank you.

File: OutOfSchool/OutOfSchool.WebApi/Services/StatusService.cs [hunk 2]
```diff
@@ -6,6 +6,7 @@
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Localization;
 using Microsoft.Extensions.Logging;
+using Nest;
```
Comment (VadymLevkovskyi) [inline OutOfSchool/OutOfSchool.WebApi/Services/StatusService.cs]: This using might be redundant. Please recheck

Comment (alrom) [reply OutOfSchool/OutOfSchool.WebApi/Services/StatusService.cs]: I did it

Comment (VadymLevkovskyi) [reply OutOfSchool/OutOfSchool.WebApi/Services/StatusService.cs]: Splendid!

**Review by VadymLevkovskyi — APPROVED** [review]

**Pull request merged**